### ISSUE TITLE
Blood vial

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#2334]
**Feature description**
A regular blood vial, it can store 10 ml of a liquid

**Feature purpose**
More liquid storage options, and it also prevents humans getting unconsious because the jar sucked up a litre of their blood

**Additional context**
none

**Comment by DindinYT37** [comment]
Why couldn't you just use a syringe instead?

**Comment by bibbybabby** [comment]
> Why couldn't you just use a syringe instead?

because its 300 ml

**Comment by DindinYT37** [comment]
> > Why couldn't you just use a syringe instead?
> 
> because its 300 ml

Yeah, but I mostly just meant what you said about humans getting unconscious.

**Comment by bibbybabby** [comment]
> > > Why couldn't you just use a syringe instead?
> > 
> > 
> > because its 300 ml
> 
> Yeah, but I mostly just meant what you said about humans getting unconscious.

3 syringes makes a human unconscious, I need 4 vials to interact with and pour into medical tech things and other people

**Comment by DindinYT37** [comment]
> > > > Why couldn't you just use a syringe instead?
> > > 
> > > 
> > > because its 300 ml
> > 
> > 
> > Yeah, but I mostly just meant what you said about humans getting unconscious.
> 
> 3 syringes makes a human unconscious, I need 4 vials to interact with and pour into medical tech things and other people

Well, those numbers are quite honestly just unfortunate, then.
